.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
       - restore_cache: *restore_cache
       - attach_workspace: {at: node_modules}
       - run: .circleci/setup_git
-      - run: yarn exec nps test
```

Comment:
nps runs the `generate` cmd which creates a cli and runs it. all for removing nps but we're losing coverage here we need to make up some other way.